addons/auth/addon/authenticators/base.js

```diff
@@ -66,6 +66,12 @@ export default class BaseAuthenticator extends SimpleAuthBaseAuthenticator {
         headers: { Authorization: `Bearer ${token}` },
       }),
     );
+
+    // Note: Always consume response object in order to avoid memory leaks.
+    // visit https://undici.nodejs.org/#/?id=garbage-collection for more info.
```

Comment:
Just want to point out we don't use `undici` since this is a node package, but is still relevant since it points out the fetch spec depends on [garbage collection](https://fetch.spec.whatwg.org/#garbage-collection) which means this behavior might not be well defined. Might be worth clarifying that in this comment?

Also, should we consider making a `HEAD` request instead as suggested in your link and [here](https://github.com/node-fetch/node-fetch/issues/83) if we don't need the body? I don't think there's much data returned so it might not matter too much either way

Reply:
thank you for the feedback!! comment updated!
i tested out using the head method tho i get the "method not allowed" error in the console :(
![image](https://github.com/user-attachments/assets/4b534cfb-567a-439d-b887-7dc8f9727c16)

Reply:
Ah I guess we don't support `HEAD` requests 😢